specs/_features/eip7594/fork-choice.md

```diff
@@ -0,0 +1,183 @@
+# EIP-7594 -- Fork Choice
+
+## Table of contents
+<!-- TOC -->
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+
+- [Introduction](#introduction)
+  - [Helpers](#helpers)
+    - [`get_custody_parameters`](#get_custody_parameters)
+    - [`get_sampling_columns`](#get_sampling_columns)
+    - [`retrieve_column_sidecars`](#retrieve_column_sidecars)
+    - [`is_data_available`](#is_data_available)
+    - [`is_chain_available`](#is_chain_available)
+    - [`get_head`](#get_head)
+    - [`is_peer_sampling_required`](#is_peer_sampling_required)
+- [Updated fork-choice handlers](#updated-fork-choice-handlers)
+  - [`on_block`](#on_block)
+    - [Pull-up tip helpers](#pull-up-tip-helpers)
+      - [`compute_pulled_up_tip`](#compute_pulled_up_tip)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+<!-- /TOC -->
+
+## Introduction
+
+This is the modification of the fork choice accompanying EIP-7594
+
+### Helpers
+
+#### `retrieve_column_sidecars`
+
+`def retrieve_column_sidecars(beacon_block_root: Root, columns_to_retrieve: Sequence[ColumnIndex]) -> Sequence[DataColumnSidecar]`
```

Comment:
This is the Electra equivalent to `retrieve_blobs_and_proofs` in Deneb and isn't meant to be implemented. I think we should delete this section and add a comment similar to how Deneb does it.

https://github.com/ethereum/consensus-specs/blob/5cce790decfb362bef300a4ca9f8075b1699ccb1/specs/deneb/fork-choice.md?plain=1#L53-L58